docs/deployment/quick_start.rst

```diff
@@ -41,11 +29,11 @@ First, generate a public and private key pair
     openssl rsa -in deepforge_keys/private_key -pubout > deepforge_keys/public_key
     export TOKEN_KEYS_DIR="$(pwd)/deepforge_keys"
 
-Then start DeepForge using docker-compose:
+Then start DeepForge using ``docker-compose run``:
 
 .. code-block:: bash
 
-    docker-compose up
+    docker-compose --file docker-compose.yml run -v "${TOKEN_KEYS_DIR}:/token_keys" -p  8888:8888 -p 8889:8889 -e "NODE_ENV=production" server
```

Comment:
I don't think we need `-e "NODE_ENV=production"` here, do we?

Reply:
I don't think we need it.